examples/kafka-hub/README.md

```diff
@@ -29,6 +29,12 @@ As mentioned above implementation is based on Kafka message broker. Therefore, t
 
 As you can see the current implementation is relying on two different technologies. An IdP is used to handle any authentication and authorization request. This would be the normal case in any production environment. For the other quality of services such as message persistence, subscription management, etc the implementation depends on Kafka message broker. 
 
+`Event Consolidator` is a supportive service which handles the race-conditions related to `topic` and `subscription` management.
+
```

Comment:
I think we should explain how we can get race conditions as well.

Reply:
Fixed with [8d94ffb0661a35177eb6e84ff436f147a3ff32fe](https://github.com/ballerina-platform/module-ballerina-websubhub/pull/557/commits/8d94ffb0661a35177eb6e84ff436f147a3ff32fe)